.github/workflows/benchmark_on_push.yml

```diff
@@ -31,12 +31,9 @@ jobs:
 
       - name: Install python dependencies
         run: |
-          python -m uv pip install --upgrade pip wheel setuptools wget cmake casadi numpy
+          python -m uv pip install --upgrade pip wheel setuptools wget numpy
```

Comment:
```suggestion
```

I don't think even this should be needed now, `asv` sets up and manages environments automatically from its config file.